app/assets/stylesheets/modules/dependencies.css

```diff
@@ -0,0 +1,114 @@
+.deps{
+  padding-left: 25px;
+  padding-top: 6px;
+  display: block;
+}
+
+.scope{
+  color: grey;
+  font-size: 15px;
+  font-weight: bold;
+  font-style: italic;
+  padding-left: 13px;
+  padding-top: 5px;
+  display: block;
+  cursor: pointer;
+  user-select: none;
+}
+
+.scope::before {
+  position: relative;
+  content: "";
+  display: inline-block;
+  width: 0.4em;
+  height: 0.4em;
+  border-right: 0.2em solid grey;
+  border-top: 0.2em solid grey;
+  transform: rotate(45deg);
+  margin-right: 0.5em;
+}
+
+.scope--expanded::before {
+  transform: rotate(135deg);
+}
+
+.scope:hover {
+  color: #e9573f;
+  transition: 0.3s;
+}
+
+.scope:hover::before {
+  color: #e9573f;
+  transition: 0.3s;
+  border-color: #e9573f;
+}
+
+.deps_toggle{
+  display: none;
+}
+
+.deps_item{
+  font-weight: bold;
+  font-size: 16px;
+  color: black;
+  cursor: pointer;
+  transition-duration: 1s;
+  transition-property: color;
+  display: inline-block;
+}
+
+.deps_item--error{
+  font-weight: normal;
+  font-style: italic;
+  color: red;
+  font-size: 15px;
+}
+
+.deps_item--details{
+  font-weight: normal;
+  font-style: italic;
+  font-size: 15px;
+}
+
+.deps_item:hover {
+  color: #e9573f;
+  transition: 0.3s;
+}
+
+.deps_expanded{
+  cursor: pointer;
+  user-select: none;
+}
+
+.deps_expanded::before{
+  content: "\25B6";
+  color: #e9573f;
+  display: inline-block;
+  margin-right: 3px;
+  transition: 0.3s;
+}
+.deps_expanded-down::before{
+  transform: rotate(90deg);
+  transition: 0.3s;
+}
+
+.caret {
+  cursor: pointer;
+  user-select: none;
+}
+
+.caret::before {
```

Comment:
`caret` and `caret::before` is same as `deps_expanded` and `deps_expanded::before` except color. You should be able to override color in `deps_expanded-down::before`. please remove this.

Reply:
Thanks I've changed it. I renamed it as deps_expanded-link because I need to use it to click for ajax requests.